src/openrct2/interface/Window_internal.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Window_internal.h"
+
+#include "../world/Sprite.h"
+
```

Comment:
Can we move this file in its entirety to the `openrct2-ui` subproject? Most (if not all) of the callees are in the ui subproject as well, right?

Reply:
Thats planned for when everything is complete and have all the private members hidden behind an interface